src/gg.regression.unity.bots/Runtime/Prefabs/RGOverlayCanvasV2.prefab

```diff
@@ -792,7 +792,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```

Comment:
This seems to now be active by default... I don't think that was intentional

Reply:
Correct - fixed in latest